### ISSUE TITLE
Expose Paint Color4f

### ISSUE THREAD

**Issue by Solido** [JetBrains/skija#31]
Hey it's me again,

Since I'm having a good time with Skija you'll see me open some tickets here & here as I stumble upon.
Maybe there are already in the roadmap and I just need to be patient. Thx !

Please expose
 ```
SkPaint green1, green2;
 green1.setColor4f({0, 1, 0, 1}, nullptr); 
```

**Comment by tonsky** [comment]
Shouldn’t be a problem, thanks!

**Comment by tonsky** [comment]
Implemented in 0.4.30
